wasm32::memory_atomic_wait32 not found

Hi,
I can't compile `parking_lot` on Wasm even when I follow the README. 

Command and error:
```
$ RUSTFLAGS="-C target-feature=+atomics" trunk serve
bad status returned from cargo build:    Compiling parking_lot_core v0.8.1 (D:\repos\parking_lot\core)
   Compiling unicode-normalization v0.1.16
   Compiling standback v0.2.13
   Compiling unicase v2.6.0
   Compiling num-traits v0.2.14
   Compiling parking_lot v0.11.0
   Compiling hashbrown v0.8.2
   Compiling indexmap v1.6.0
   Compiling bytes v0.4.12
error[E0425]: cannot find function `memory_atomic_wait32` in module `wasm32`
  --> D:\repos\parking_lot\core\src\thread_parker\wasm_atomic.rs:76:29
   |
76 |             let r = wasm32::memory_atomic_wait32(self.ptr(), PARKED, -1);
   |                             ^^^^^^^^^^^^^^^^^^^^ not found in `wasm32`
```

It looks like it's not possible to compile `parking_lot` on Wasm at all if you use `rustup` to manage your Rust:
> This intrinsic is only available when the standard library itself is compiled with the atomics target feature. This version of the standard library is not obtainable via rustup, but rather will require the standard library to be compiled from source.

- From https://doc.rust-lang.org/core/arch/wasm32/fn.memory_atomic_wait32.html#availability

---
- `rustc -V` - `rustc 1.50.0-nightly (1c389ffef 2020-11-24)`
- Windows 10

---

The simplest solution would be probably a README update.
Thanks!

Thanks for the super quick answer.
I had the latest nightly, the problem was missing `-Zbuild-std` parameter (yeah, without the space between `Z` and `build`).

However the code:
```rust
...
Mutex::new(0);
...
```
fails now with
```rust
error[E0277]: the size for values of type `parking_lot::RawMutex` cannot be known at compilation time
   --> src\lib.rs:7:5
    |
7   |     Mutex::new(0);
    |     ^^^^^^^^^^ doesn't have a size known at compile-time
```
and dozens errors like `error: duplicate lang item in crate core: RangeFrom`.

But I don't have a motivation to resolve them, my original problem was that `parking_lot` silently fails on stable Rust and the Wasm app that uses `parking_lot` as a dependency fails in runtime with a non-expressive error `Uncaught TypeError: Failed to resolve module specifier "env". Relative references must start with either "/", "./", or "../".`. But it's probably a Rust compiler or `wasm-bindgen` issue.

Thank a lot for the help!
Cheers, Martin

--- a/README.md
+++ b/README.md
@@ -87,9 +87,9 @@ lock.
 There are a few restrictions when using this library on stable Rust:
 
 - The `wasm32-unknown-unknown` target is only fully supported on nightly with
-  `-C target-feature=+atomics` in `RUSTFLAGS` and `-Z build-std` passed to cargo.
-  parking_lot will work mostly fine on stable, the only difference is it will
-  panic instead of block forever if you hit a deadlock.
+  `-C target-feature=+atomics` in `RUSTFLAGS` and `-Zbuild-std=panic_abort,std`
+  passed to cargo. parking_lot will work mostly fine on stable, the only
+  difference is it will panic instead of block forever if you hit a deadlock.
   Just make sure not to enable `-C target-feature=+atomics` on stable as that
   will allow wasm to run with multiple threads which will completely break
   parking_lot's concurrency guarantees.

